src/Shared/Recruit.Vacancies.Client/Infrastructure/Client/QaVacancyClient.cs

```diff
@@ -81,20 +81,29 @@ public async Task<QaDashboard> GetDashboardAsync()
             return dashboard;
         }
 
-        public async Task<List<VacancyReview>> GetSearchResultsAsync(string searchTerm)
+        public async Task<VacancyReview> GetSearchResultAsync(string searchTerm)
         {
             //The return type is a list to allow adding results from other searches to be implemented in future
             var result = new List<VacancyReview>();
```

Comment:
Kill this and the comment above.

Reply:
👍

Reply:
don't kill the comment... it is important as otherwise it does not make sense.